specs/NNNN-template/README.md

```diff
@@ -0,0 +1,161 @@
+
+# VEP-NNNN: Your short, descriptive title
+
+* **Author(s):** Name (mail), ...
+* **Status:** draft | implemented | rejected | withdrawn | replaced
+
+To get started with this template:
+
+- [ ] **Create an issue in Github (if one does not exists already)**
+- [ ] **Make a copy of this template directory.**
+  Copy this template into the specs directory and name it
+  `NNNN-short-descriptive-title`, where `NNNN` is the issue number (with no
+  leading-zero padding) created above.
+- [ ] **Fill out this file as best you can.**
+  There are instructions as HTML comments.
+  At minimum, you should fill in the "Summary" and "Motivation" sections.
+- [ ] **Create a PR for this VEP.**
+- [ ] **Merge early and iterate.**
```

Comment:
Maybe, we could also add a step for creating a "Milestone" for better progress tracking?

Reply:
Can you elaborate on the idea?

Reply:
Well, I was thinking about something similar to what we did with the Execution API logs, https://github.com/vmware/versatile-data-kit/milestone/1?closed=1

Reply:
We do not have it yet as a required process for all features. Though I like it. It's up to the feature lead. In the spec there is section "Implementation stories" which can just contain link to that milestone.